crates/transaction-pool/src/error.rs

```diff
@@ -29,6 +30,15 @@ pub enum PoolError {
     /// respect the max_init_code_size.
     #[error("[{0:?}] Transaction's size {1} exceeds max_init_code_size {2}.")]
     TxExceedsMaxInitCodeSize(TxHash, usize, usize),
+    /// Thrown if a replacement transaction's gas price is below the already imported transaction
+    #[error("[{0:?}]: insufficient gas price to replace existing transaction.")]
+    AccountNotFound(TxHash),
```

Comment:
this is not named intuitively.
this would be `ReplacementTransactionUnderpriced` error instead.

Reply:
I should've changed the doc for this. This should mirror [geth's TxPool error](https://github.com/ethereum/go-ethereum/blob/5d8bff1d7261aed1a68235b0223d44a4fb4fb259/core/txpool/txpool.go#L64)